docs/source/index.rst

```diff
@@ -55,83 +55,94 @@ processo de submissão e comunidade de usuários.
+  do processo de geração.
 
 
 .. _journal-meta-csv:
 
-Metadados dos periódicos
-^^^^^^^^^^^^^^^^^^^^^^^^
+Metadados de periódicos
+^^^^^^^^^^^^^^^^^^^^^^^
 
-Adicionalmente, diversos metadados dos periódicos necessários para a identificação de 
-elementos em ``<journal-meta>`` estão disponíveis em uma listagem no formato :term:`csv`, 
-que pode ser baixada `aqui <http://static.scielo.org/sps/titles-tab-utf-8.csv>`_. 
-Esta listagem é atualizada semanalmente, às terças-feiras.
+Complementarmente, encontra-se disponível em formato :term:`csv` uma lista de
+metadados de periódicos necessários para identificação de elementos em
+``<journal-meta>``. O documento pode ser baixado a partir deste `link
+<http://static.scielo.org/sps/titles-tab-utf-8.csv>`_ e sua atualização é
+semanal, sempre às terças-feiras.
 
 
 Convenções utilizadas neste guia
 --------------------------------
 
-A fim de facilitar a compreensão deste guia, foram utilizadas algumas
-convenções de estilo e formatação.
+Para facilitar a compreensão deste guia, foram utilizadas algumas convenções de
+estilo e formatação.
 
-.. note:: Estas caixas apresentam informação importante e diretamente relacionada ao contexto
-          em que estão inseridas.
+Formatação
+^^^^^^^^^^
 
+.. note:: Estas caixas apresentam informação importante e diretamente
+          relacionada ao contexto em que estão inseridas.
 
 *Itálico*
-  Utilizado para nomes de arquivos, normas, URLs, referências ativas a elementos do XML 
-  (hiperlinks) ou para introduzir novos termos.
-
-**Negrito**
-  Utilizado para identificar textos que devem ser substituídos por valores fornecidos
-  pelo usuário.
-
-``Largura fixa``
-  Utilizado para exemplos, trechos ou referências estáticas a elementos ou atributos
-  do XML.
-
-
-Para cada elemento descrito, há uma lista de definições no formato::
+  Utilizado sempre para:
+  * palavras e/ou termos estrangeiros;
+  * nomes de publicações, empresas, instituições, projetos, técnicas,
+    tecnologias, marcas etc;
+  * nomes de arquivos, normas e referências.
 
-  Aparece em
-    <journal-meta>
-
-  Atributos obrigatórios
-    @journal-id-type
-
-  Ocorre
-    Uma vez
-
-
-Onde:
-
-* **Aparece em**: Apresenta o contexto (elemento-pai) onde o elemento em foco é comumente 
-  identificado. O fato de um determinado contexto não constar nesta lista não 
-  o exclui automaticamente, e a especificação JATS Publishing deve ser consultada.
-* **Atributos obrigatórios**: Apresenta apenas os atributos onde, no caso da 
-  presença do elemento, devem ser marcados obrigatoriamente. Esta definição pode vir 
-  acompanhada de seus valores, também obrigatórios. Caso não existam atributos 
-  obrigatórios, este item é omitido.
-* **Ocorre**: O número de vezes que o elemento pode ocorrer em um contexto.
+*``Itálico em fonte monoespaçada``*
+  Identifica palavras ou termos que devem ser substituídos por conteúdo adequado
```

Comment:
Me parece que o texto original se perdeu... Este trecho se refere ao negrito. Ficou similar no trecho do negrito e do itálico. 
O original do itálico é:
"Utilizado para nomes de arquivos, normas, URLs, referências ativas a elementos do XML (hiperlinks) ou para introduzir novos termos."